### PR TITLE
Generate pom for aggregate javadocs and clean up root pom

### PR DESCRIPTION
**Pull request by srnagar**

### CODE REVIEW

File: eng/scripts/aggregate_javadoc_configuration.txt [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# File containing frequently updated aggregate JavaDoc configuration
+
+# Adds the following groups to the aggregate JavaDoc
+# Group;Title;Package
```
Comment (alzimmermsft) [inline eng/scripts/aggregate_javadoc_configuration.txt:4]: Future enhancement, provide a utility that can update these automatically. Should be able to do something where the module-info of a library is inspected for exports, not exports to, and create the colon delimited list of packages included in Javadocs.

File: eng/scripts/aggregate_javadoc_configuration.txt [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+Group;Azure Web Pub Sub;com.azure.messaging.webpubsub*
+
+# Adds the following links to external javadocs
+# Link;URL 
+Link;https://docs.oracle.com/javase/8/docs/api/
+Link;https://projectreactor.io/docs/core/release/api/
+Link;https://projectreactor.io/docs/netty/release/api/
+Link;https://fasterxml.github.io/jackson-annotations/javadoc/2.12/
+Link;https://fasterxml.github.io/jackson-core/javadoc/2.12/
+Link;https://fasterxml.github.io/jackson-databind/javadoc/2.12/
```
Comment (alzimmermsft) [inline eng/scripts/aggregate_javadoc_configuration.txt:64]: Need to be updated to 2.13 but this isn't an issue for this PR, just a reminder for ourselves

**Review by alzimmermsft — APPROVED** [review]

File: eng/scripts/aggregate_javadoc_configuration.txt (hunk 1 above)
Comment (srnagar) [reply eng/scripts/aggregate_javadoc_configuration.txt:4]: Created https://github.com/Azure/azure-sdk-for-java/issues/30131

File: eng/scripts/aggregate_javadoc_configuration.txt (hunk 2 above)
Comment (srnagar) [reply eng/scripts/aggregate_javadoc_configuration.txt:66]: Will have a follow-up PR for updating the version.

**Pull request merged**